### PR TITLE
Add a warning that the quiz is WIP

### PR DESCRIPTION
**Pull request by WaffleLapkin**
Since the quiz seems to be shared publicly, we should probably warn people that it's very much WIP.

This should be merged after #17 and #18, so that every page has warnings.

### CODE REVIEW

File: src/include/quiz-is-wip.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+> **Warning:** this quiz is still "work-in-progress", some questions might not have good explanations (or any at all),
+> formatting/structure/titles/etc are not final, and so on.
+> You might want to return here on a later date.
```
Comment (WaffleLapkin) [inline src/include/quiz-is-wip.md:3]: I don't really like this wording, but I couldn't come up with something good...

Comment (Nilstrieb) [reply src/include/quiz-is-wip.md:3]: yeah it can be improved but i'm not sure how to best word it

**Comment by Nilstrieb** [comment]
WIP (waffle in progress)

**Comment by WaffleLapkin** [comment]
wip (meoW in progress)

**Pull request merged**